packages/block-library/src/common.scss

```diff
@@ -154,3 +154,10 @@ html :where(img[class*="wp-image-"]) {
 	height: auto;
 	max-width: 100%;
 }
+
+/**
+ * Reset user agent styles for figure element margins.
+ */
+figure {
```

Comment:
If need be, we could always reduce the specificity of this one via `:where(figure)` if it has potential to conflict with themes' provided styles?